### DISCUSSION TITLE
Shooting for a miracle

### DISCUSSION THREAD

**Discussion by loredous** [mdubinko/gsoc-contrib#17] [General]
In seeing the renewed community interest in this project, I have been looking back at some of the thoughts I had previously around ways that the community might be able to breathe a bit more life into the GSOC, since it seems to be mostly forgotten by Xiegu, and I decided I would try for a shot in the dark. 

I am going to see if I can initiate any sort of communication with anyone at Xiegu, with hopes of convincing them that even if they can't or won't release new firmware, that there is an amateur radio operator community here that potentially could create and maintain a community-supported firmware. I don't have high expectations, but sometimes the world surprises you. I'll keep this discussion updated with any contact I have with them.

Jeremy - K0JLB

**Comment by 2E0WKS** [comment]
I've mine running  debian from SD card and functioning gsoc app, it isn't too much of a stretch to install WiFi, install lsusb and pci tools and see what is switched on in uboot and accessable. The connectors on edge of board shown look to be for 2.ghz slot antenna.
Creating an image not that difficult as I modified pi-star with a touch screen and touch keyboard.
Kind of surprised it's not well and truly hacked for everyone.
